Userland/Libraries/LibJS/Tests/builtins/ArrayBuffer/ArrayBuffer.options.js

```diff
@@ -0,0 +1,42 @@
+test("construct with options", () => {
+    expect(new ArrayBuffer(5, { maxByteLength: 5 })).toBeInstanceOf(ArrayBuffer);
+
+    expect(new ArrayBuffer(1).resizable).toEqual(false);
+    expect(new ArrayBuffer(1, { maxByteLength: 1 }).resizable).toEqual(true);
```

Comment:
These tests are a little more clear but could also be converted to `describe("construct with options")` and then some `test("correct results", () => {})` and test("throws if byte length exceeds max byte length", () => {})` etc.

(See also my other comment about the tests on the next commit)

Reply:
Additionally, I think these should be moved to `ArrayBuffer.js`; we usually name the file according to the function only (so no `options` suffix).